haskell/src/Lentille/GraphQL.hs

```diff
@@ -129,4 +161,5 @@ streamFetch client mkArgs transformResponse = go Nothing
       unless (null decodingErrors) (stopLentille $ DecodeError decodingErrors)
 
       -- TODO: implement throttle
```

Comment:
I think you can remove the TODO

Reply:
Python implementation got 1s delay between each API calls: https://github.com/change-metrics/monocle/blob/master/monocle/github/graphql.py#L86
I guess we will need more experimentation about crawling large repositories to see how this graphQL crawler implementation behaves with Github.